src/panels/lovelace/tile-features/hui-vacuum-commands-tile-feature.ts

```diff
@@ -85,7 +85,7 @@ export const VACUUM_COMMANDS_BUTTONS: Record<
           translationKey: "start",
           icon: mdiPlay,
           serviceName: "start",
-          disabled: canStart(stateObj),
+          disabled: !canStart(stateObj),
```

Comment:
This is still wrong, the button would now be enabled when the state === unavailable

Reply:
True and the button linked to `canReturnHome()` has the same issue.

Reply:
Actually, that is not true. If the entity is not available all buttons are disabled by default in `render()`.

Reply:
The unavailable condition is checked for every button line 185.